### PR TITLE
staging/olm: Re-generate test fake package(s)

### PR DESCRIPTION
**Pull request by timflannagan**
Run `go generate ./...` at the root of the
staging/operator-lifecycle-manager repository.

### CODE REVIEW

**Comment by timflannagan** [comment]
Need to verify that the unit-olm check is green before merging.

/hold

**Comment by timflannagan** [comment]
Looks like we're good to go

/hold cancel

**Review by ankitathomas — APPROVED** [review]
/lgtm

**Pull request merged**
